examples/getting-started/blocks/package.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
This `blocks/` directory is a [workspace](https://docs.npmjs.com/cli/v7/using-npm/workspaces), with its own `package.json`, so that the [connector plugin](https://github.com/kienstra/headless-blocks/blob/33c40fe4a22f59fe28983087c5e450e759a69acf/js/src/helpers/addPreview.ts#L9) can `import` these blocks.

Then, the connector plugin [filters](https://github.com/studiopress/genesis-custom-blocks/blob/10de2822a0c7260ba159e978b23eb2aba6a1302c/js/src/block-editor/components/preview.js#L36) the preview component that Genesis Custom Blocks renders:

<img width="832" alt="here-is-the" src="https://user-images.githubusercontent.com/4063887/109490787-55c5db80-7a4e-11eb-89d6-9b038c020c61.png">